pycodestyle.py

```diff
@@ -119,6 +119,7 @@
 OPERATOR_REGEX = re.compile(r'(?:[^,\s])(\s*)(?:[-+*/|!<=>%&^]+)(\s*)')
 LAMBDA_REGEX = re.compile(r'\blambda\b')
 HUNK_REGEX = re.compile(r'^@@ -\d+(?:,\d+)? \+(\d+)(?:,(\d+))? @@.*$')
+NOT_DUNDER_ALL_REGEX = re.compile(r'^__(?!all__)')
```

Comment:
I think we would want to exclude quite a lot more, where the line isnt simply storing metadata, like , attribute access (`.`), indexing (`[]`) , etc